src/Asset/Capacity/AbstractCapacity.php

```diff
@@ -65,6 +66,28 @@ public function getSpecificRights(): array
         return [];
     }
 
+    public function canBeDisabled(CommonDBTM $asset, string $table = null): bool
```

Comment:
You should not add more params than the interface declare. If you want to have a generic counting method, then you can declare a `protected function countUsages(...)` method.

Also, I do not think that we should provide a default implementation for this method.

By the way, please try to always pass itemtypes instead of tables in parameters.